### PR TITLE
Package biocaml.0.10.0

### PR DESCRIPTION
**Pull request by pveber**
### `biocaml.0.10.0`
The OCaml Bioinformatics Library
Biocaml aims to be a high-performance user-friendly library for
Bioinformatics. See the [website](http://biocaml.org) for detailed API
documentation. We welcome contributors and feedback from users. Please
contact us with any comments and suggestions for features you would
like added.

---
* Homepage: http://biocaml.org
* Source repo: git+https://github.com/biocaml/biocaml.git
* Bug tracker: https://github.com/biocaml/biocaml/issues

---
:camel: Pull-request generated by opam-publish v2.0.0

### CODE REVIEW

File: packages/biocaml/biocaml.0.10.0/opam [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  "David Koppstein"
+  "Alexander Gryzlov"
+  "Francois Berenger"
+]
+
+build: ["dune" "build" "-p" name "-j" jobs]
+
+depends: [
+  "base64"
+  "dune" {build}
```
Comment (avsm) [inline packages/biocaml/biocaml.0.10.0/opam]:
```suggestion
  "dune"
```

Comment (pveber) [reply packages/biocaml/biocaml.0.10.0/opam]: This is to enforce rebuild of the library when dune version is changed? In any case, I'll gladly follow your suggestion.

**Comment by avsm** [comment]
Thanks! You may want to announce this on <https://discuss.ocaml.org>, where we have a Community category and an `announce` tag for this purpose.

**Comment by pveber** [comment]
Thanks a lot @avsm!

**Pull request merged**